mypy.ini

```diff
@@ -270,7 +271,31 @@ ignore_missing_imports = True
 ignore_missing_imports = True
 
 [mypy-torch._inductor.*]
-ignore_errors = True
+disallow_any_generics = True
 
 [mypy-torch._dynamo.*]
-ignore_errors = True
+disallow_any_generics = True
+
+[mypy-colorama.*]
```

Comment:
I missed this in my first approval, but colorama does actually have a `type-package`

Reply:
OK, I'll try removing it in a subsequent PR